### ISSUE TITLE
"PropertySpelling: Unknown property text" false negative

### ISSUE THREAD

**Issue by berbaquero** [brigade/scss-lint#281]
In Sass it's possible to do:

``` scss
.stuff {
  font: {
    family: Arial;
    size: 2em;
  }
}
```

And it would output:

``` css
.stuff {
  font-family: Arial;
  font-size: 2em;
}
```

But if I want to do that with `text`, as in:

``` scss
.stuff {
  text: {
    transform: uppercase;
    align: center;
  }
}
```

The linter throws `PropertySpelling: Unknown property text`.

Maybe I'm missing something, but it should not throw this warning, right?
I understand it works with `font` particularly, because `font` it's a shorthand property on it's own.
